tests/groups.rs

```diff
@@ -18,13 +18,36 @@ USAGE:
 
 For more information try --help";
 
+#[allow(unused)]
 static REQ_GROUP_CONFLICT_REV: &str = "error: The argument '--delete' cannot be used with '<base>'
 
 USAGE:
     clap-test <base|--delete>
 
 For more information try --help";
 
+static REQ_GROUP_CONFLICT_ONLY_OPTIONS: &str =
+    "error: Found argument '--all' which wasn't expected, or isn't valid in this context
+
+If you tried to supply `--all` as a PATTERN use `-- --all`
+
+USAGE:
+    clap-test <-a|--delete>
+
+For more information try --help";
+
+// FIXME: This message has regressed after https://github.com/clap-rs/clap/pull/1856
```

Comment:
Can you please create an issue for fixing this?

Reply:
Please don't merge without doing this

Reply:
https://github.com/clap-rs/clap/issues/1859
https://github.com/clap-rs/clap/issues/1859